devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -79,16 +76,16 @@ def parse(String description) {
 	} else {
 		def cluster = zigbee.parse(description)
 		
-		if (cluster && cluster.clusterId == ON_OFF_CLUSTER_VALUE && cluster.command == ON_OFF_COMMAND_VALUE) {
+		if (cluster && cluster.clusterId == ON_OFF_CLUSTER && cluster.command == CONFIGURE_REPORTING_RESPONSE) {
 			if (cluster.data[0] == 0x00) {
 				sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 			}
 		} else {
 			def zigbeeMap = zigbee.parseDescriptionAsMap(description)
 			
-			if (zigbeeMap.clusterInt ==  MOTION_CLUSTER_VALUE && zigbeeMap.attrInt == MOTION_STATUS_VALUE) {
+			if (zigbeeMap.clusterInt ==  MOTION_CLUSTER && zigbeeMap.attrInt == MOTION_STATUS_ATTRIBUTE) {
 				def childDevice = getChildDevices()?.find {
-					it.device.deviceNetworkId == "${device.deviceNetworkId}:1" 
+					it.device.deviceNetworkId == addChildSensor() 
```

Comment:
In such case, almost every incoming message would try to create a new child device.